deployment/src/main/java/org/wso2/testgrid/deployment/deployers/ShellDeployer.java

```diff
@@ -89,7 +93,21 @@ public DeploymentCreationResult deploy(TestPlan testPlan,
         }
 
         result.setName(deploymentPatternConfig.getName());
-        result.setHosts(infrastructureProvisionResult.getHosts());
+
+        List<Host> hosts = new ArrayList<>();
+        Host tomcatHost = new Host();
+        tomcatHost.setLabel("tomcatHost");
+        tomcatHost.setIp("ec2-52-54-230-106.compute-1.amazonaws.com");
+        Host tomcatPort = new Host();
+        tomcatPort.setLabel("tomcatPort");
+        tomcatPort.setIp("8080");
+        hosts.add(tomcatHost);
+        hosts.add(tomcatPort);
+
+        DeploymentCreationResult result1 = DeploymentUtil.getDeploymentCreationResult(infrastructureProvisionResult
+                .getDeploymentScriptsDir());
```

Comment:
What's the reason to create another DeploymentCreationResult instance? Can't we use `result` instance?

Reply:
Yes, we can. Made changes in e8bb16e